rs/bitcoin/kyt/btc_kyt_canister.did

```diff
@@ -64,6 +67,12 @@ type BtcNetwork = variant {
     testnet
 };
 
+type KytMode = variant {
+    AcceptAll;
+    RejectAll;
+    Normal;
```

Comment:
`NormalOperation`?

Reply:
This was copied from the existing KYT canister. So I am inclined to keep it the same.